evap/evaluation/templatetags/morefilters.py

```diff
@@ -70,3 +70,7 @@ def render(self, context):
 @register.filter
 def is_false(arg): 
     return arg is False
+
+@register.filter
+def is_choice_field(field):
+    return field.field.__class__.__name__ == "TypedChoiceField"
```

Comment:
here it's a different situation than in the other case. from the ids and names of the questions i can't see whether they need a textfield or a choice field. so i have to look that up somehow else.